### PR TITLE
Add an old chinese story about recursion.

### PR DESCRIPTION
**Pull request by wqfeng**

### CODE REVIEW

**Comment by wqfeng** [comment]
@cbuckey-uda , I just added an old Chinese story about recursion. You might find it fun to merge it to your master branch. :)

**Comment by wqfeng** [comment]
Opps! I should @casey-collab . You are a robot, aren't you?

File: Chinese/monk/monk.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+从前有座山
```
Comment (casey-collab) [inline Chinese/monk/monk.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by wqfeng** [comment]
@casey-collab you are clever enough! I link the file with my latest commit. Go have a check!

File: Chinese/monk/monk.md (hunk 1 above)
Comment (casey-collab) [inline Chinese/monk/monk.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by wqfeng** [comment]
So you've merged? I will go to the forked repository and have a check. Thanks @casey-collab !

File: Chinese/monk/monk.md (hunk 1 above)
Comment (casey-collab) [inline Chinese/monk/monk.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by wqfeng** [comment]
@casey-collab it seems you have a bug :( @cbuckey-uda instead

File: Chinese/monk/monk.md (hunk 1 above)
Comment (casey-collab) [inline Chinese/monk/monk.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: Chinese/monk/monk.md (hunk 1 above)
Comment (wqfeng) [inline Chinese/monk/monk.md]: @casey-collab I've fixed that. Go ahead to merge it.

**Comment by wqfeng** [comment]
@casey-collab please have another try.

File: Chinese/monk/monk.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
```
Comment (casey-collab) [inline Chinese/monk/monk.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: Chinese/monk/monk.md (hunk 1 above)
Comment (wqfeng) [inline Chinese/monk/monk.md]: @casey-collab I do link that page. Please have another try.

**Comment by wqfeng** [comment]
@casey-collab please have another try.

File: Chinese/monk/monk.md (hunk 2 above)
Comment (casey-collab) [inline Chinese/monk/monk.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Sorry @wqfeng, Casey can't handle filenames with non-ascii characters. This looks good to go! :umbrella:

**Pull request merged**